pkg/services/sqlstore/team.go

```diff
@@ -157,14 +186,18 @@ func SearchTeams(query *models.SearchTeamsQuery) error {
 	var sql bytes.Buffer
 	params := make([]interface{}, 0)
 
+	filteredUsers := getFilteredUsers(query.SignedInUser, query.HiddenUsers)
 	if query.UserIdFilter > 0 {
-		sql.WriteString(getTeamSearchSqlBase())
+		sql.WriteString(getTeamSearchSqlBase(filteredUsers))
 		params = append(params, query.UserIdFilter)
 	} else {
-		sql.WriteString(getTeamSelectSqlBase())
+		sql.WriteString(getTeamSelectSqlBase(filteredUsers))
+	}
+	for _, user := range filteredUsers {
+		params = append(params, user)
 	}
```

Comment:
```suggestion
	params = append(params, filteredUsers...)
```

Reply:
This is not compiling: `cannot use filteredUsers (type []string) as type []interface {} in append`